test/tests/misc/solution_invalid/gold/solution_invalid_warning.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app_name": "main",
+    "current_time": "Thu Jul 11 10:35:37 2024",
+    "executable": "/Users/lim2/Research/MOOSE/framework/worktrees/solutioninvalid_warning/test/tests/misc/solution_invalid/../../../moose_test-opt",
```

Comment:
let's get rid of all these lines by adding
```
execute_system_information_on = none
```
to your JSON output block in the test